src/oscar/apps/dashboard/catalogue/forms.py

```diff
@@ -27,17 +29,31 @@
 
 BaseCategoryForm = movenodeform_factory(
     Category,
-    fields=['name', 'slug', 'description', 'image', 'is_public'],
+    fields=['name', 'slug', 'description', 'meta_title', 'meta_description', 'image', 'is_public'],
     exclude=['ancestors_are_public'])
 
 
-class CategoryForm(BaseCategoryForm):
+class _SEOFormMixin(object):
+    def visible_fields(self):
+        return [field for field in self if not field.is_hidden and not self.is_seo_field(field)]
+
+    def seo_fields(self):
+        return [field for field in self if self.is_seo_field(field)]
+
+    @staticmethod
+    def is_seo_field(field):
```

Comment:
It is better to explicitly define on the class itself which fields are considered SEO fields, so that projects can override.